barretenberg/cpp/src/barretenberg/polynomials/polynomial.cpp

```diff
@@ -11,6 +7,13 @@
 #include <unordered_map>
 #include <utility>
 
+#include "barretenberg/common/assert.hpp"
+#include "barretenberg/common/slab_allocator.hpp"
+#include "barretenberg/common/thread.hpp"
+#include "barretenberg/common/thread_utils.hpp"
+#include "polynomial.hpp"
+#include "polynomial_arithmetic.hpp"
```

Comment:
maybe ordered incorrectly, if based on ascii

Reply:
I just use whatever comes out of the formatter.